seismic/receiver_fn/rf_quality_filter.py

```diff
@@ -13,8 +13,8 @@
 from multiprocessing import Process, Manager
```

Comment:
Cleaned up legacy use of pd.DataFrame, and better separation between processing function and post-processing handling.